### DISCUSSION TITLE
2005/05/31/%E6%88%91%E6%98%A8%E6%99%9A%E7%9A%84%E6%A2%A6

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#33] [Announcements]
# 2005/05/31/%E6%88%91%E6%98%A8%E6%99%9A%E7%9A%84%E6%A2%A6

我昨晚梦见外星人入侵地球了。

https://blog.qizhen.xyz/2005/05/31/%E6%88%91%E6%98%A8%E6%99%9A%E7%9A%84%E6%A2%A6

**Comment by ruanqizhen** [comment]
yoyo
2005年06月9日 19:33 编辑
还真是看多了

回复
哈米の哟哟
2005年05月31日 22:57 编辑
A片看多了，American movie 我也喜欢看的
